src/components/addQuestion/index.js

```diff
@@ -0,0 +1,69 @@
+import { connect } from 'react-redux';
+import { createQuestion } from '../../store/question.js';
+
+import './index.css';
+
+let AddQuestionForm = ({ onSave, positionClass }) => {

```

Comment:
In general, we should avoid using `let` and reassignment if the variable means two different things. In this case, we start with the unconnected form, and then reassign the same variable to the connected form, which is a different thing (the composition of the unconnected form with the data connections).